SpecTools/Classes/Find/Find+UIViewController.swift

```diff
@@ -21,22 +21,18 @@ extension SpecFind where T: UIViewController {
     }
     
     public func has(childViewController viewController: UIViewController) -> Bool {
-        // TODO: Check if views are added
-        // TODO: Check if childViewControllers contain viewController
-        return false
+        return element.childViewControllers.contains(viewController)
     }
     
     public func contains<T>(siblingClassInNavigationStack viewControllerClass: T.Type) -> Bool where T: UIViewController {
         guard let nc = element.navigationController else {
             return false
         }
-        // TODO: Check if navigationController.viewControllers contain any class of type T
-        return false
+        return nc.viewControllers.contains(where: { type(of: $0) == viewControllerClass })
     }
     
     public func contains<T>(childViewControllerClass viewControllerClass: T.Type) -> Bool where T: UIViewController {
-        // TODO: Check if childViewControllers contain any class of type T
-        return false
+        return element.childViewControllers.contains(where: { type(of: $0) == viewControllerClass })
```

Comment:
maybe we should check for the views here too? or at least create an input variable flag to allow devs to enable the checks

Reply:
same as above, didn't sound like this was a function checking for UIView instances, just UIViewController